mobile/src/components/BottomNavBar.tsx

```diff
@@ -1,20 +1,33 @@
 import { Pressable, View } from "react-native";
 import Ionicons from "@expo/vector-icons/Ionicons";
+import { styled } from 'nativewind';
+
+const StyledView = styled(View);
 
 export default function BottomNavBar() {
+
+  //# region Styles
+
   const pressStyle =
     "active:rounded-full active:bg-indigo-800/70 active:opacity-95 h-10 w-20 items-center justify-center";
+  const iosStyle = "w-full py-10 bg-slate-200 flex-row items-center justify-around absolute bottom-0 rounded-tl-lg rounded-tr-lg h-12";
+  const androidStyle = "w-full bg-slate-200 flex-row items-center justify-around absolute bottom-0 rounded-tl-lg rounded-tr-lg h-12";
+
+  //# endregion
+
   return (
-    <View className="w-full bg-slate-200 flex-row items-center justify-around absolute bottom-0 rounded-tl-lg rounded-tr-lg h-12">
+    <StyledView className={`
+      ios: ${iosStyle}
+      android: ${androidStyle}`}>
       <Pressable className={pressStyle} onPress={() => {}}>
-        <Ionicons name="home" size={27} color="black" />
```

Comment:
so I know for sure that Dani changed the size here too, I am just gonna write this here to remind you that there are merge conflicts

Reply:
Yeah I figured, just making sure that the actual placement of the search bar is good, as well as the clean up of styles/pages not being used. After that I can work on the conflicts.